webapp/src/main/java/com/plhal/ares/webapp/AresApiApplication.java

```diff
@@ -4,12 +4,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 
-@SpringBootApplication(scanBasePackages = {"com.plhal.ares.webapp", "com.plhal.ares.service", "com.plhal.ares.model"})
+@SpringBootApplication
```

Comment:
Sem prosim `@Import`. Ma to spise dokumentacni charakter, protoze stejne "scanujete" cely projekt a `Configuration` se vykona. Nicmene tim importem i zpusobite to, ze bude existovat jen jeden kontext se vsema beanama a ne vice oddelenych kontextu - i kdyz aplikacni dosah to v tomto pripade nema.